parachains/runtimes/starters/seedling/src/lib.rs

```diff
@@ -68,7 +68,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_version: 9320,
 	impl_version: 0,
 	apis: RUNTIME_API_VERSIONS,
-	transaction_version: 1,
+	transaction_version: 2,
```

Comment:
version bump for shell runtime is missing. 
also noticed that starters runtime is missing in the release template.

Reply:
Shell does not seem to change mach https://github.com/paritytech/cumulus/issues/1844#issuecomment-1309034509
and should be fine without bump

Reply:
> version bump for shell runtime is missing.

Not required

> also noticed that starters runtime is missing in the release template.

They used to be, they are now excluded on purpose to avoid bloating the release notes with starter runtimes that barely change. They are however rebuilt and provided as artifacts in each release.